packages/appstore/src/components/modals/real-wallets-upgrade/components/wallet-link/wallet-link-wrapper.tsx

```diff
@@ -21,50 +23,38 @@ const WalletLinkWrapper = observer(({ wallet_details, account_list }: TWalletLin
     const { is_mobile } = ui;
     return (
         <div className='wallet-link-wrapper'>
-            <div className='wallet-link-wrapper__accounts'>
-                {is_mobile && (
-                    <Text
-                        as='div'
-                        className='wallet-link-wrapper__title-text wallet-link-wrapper__accounts-title'
-                        color='prominent'
-                        size='xxxs'
-                    >
-                        <Localize i18n_default_text='Your current trading account(s)' />
-                    </Text>
-                )}
-                {account_list.map(account => {
-                    return (
-                        <WalletAccount
-                            key={`${account.account_name}-${account.currency}`}
-                            balance={account.balance}
-                            currency={account.currency}
-                            icon={account.icon}
-                            name={account.account_name}
-                        />
-                    );
-                })}
-            </div>
-            <div className='wallet-link-wrapper__link'>
-                <div
-                    className={classNames('wallet-link-wrapper__link-bracket', {
-                        'wallet-link-wrapper__link-bracket--single': account_list.length === 1,
-                    })}
-                />
-                <div className='wallet-link-wrapper__link-icon'>
-                    <Icon icon='IcAppstoreWalletsLink' size={40} />
+            <div className='wallet-link-wrapper__left'>
+                <div className='wallet-link-wrapper__accounts'>
+                    <LeftContentWithLink>
+                        {account_list.map(account => {
+                            return (
+                                <WalletAccount
+                                    key={`${account.account_name}-${account.currency}}`}
+                                    balance={account.balance}
```

Comment:
Hi @mahdiyeh-deriv, the data in the `account` object is mocked in `packages/appstore/src/constants/mock_wallet_migration_response.ts`. Since I am not utilizing the APIs to render the steps, I think it would be okay to keep using the `formatMoney` for now and later we can change it to `display_balance` while integrating the APIs.